modules/ROOT/pages/fd-reuse-config.adoc

```diff
@@ -0,0 +1,65 @@
+= Create a Connection for Reuse by Yourself or Others
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+You can create, change and reuse a connection in a Flow Designer project instead of adding your authentication information manually. Once a reusable connection is created, it is available for all Flow Designer projects.
+
+You can reuse connections:
+
+* If you are a system admin, you can enable non-admins to connect to your system using administrative authentication. This is important if the integration requires administrative system permissions.
```

Comment:
I have never heard the term "administrative authentication".  I think people ordinarily use "administrative credentials" or "an administrative connection."

Reply:
I was using the term authentication because it looks like it is used throughout most of the FD docs. What about 'your authentication'?